src/main/java/org/klodnicki/service/BookService.java

```diff
@@ -39,4 +42,25 @@ public void update(BookInfo bookInfo) {
     public List<BookInfo> findBooksByTitleAndAuthor(String title, String author) {
         return bookRepository.findBooksByTitleAndAuthor(title, author);
     }
+
+    public List<String> prepareListOfBorrowedBooksByAccount (Account account) {
```

Comment:
This method and prepareListOfBooks(Account account) should be changed. This method is not serving purpose, you can just directly use method prepareListOfBooks if you make it public. The other problem is that prepareListOfBooks doesn't only prepare, but also search for the books in database which should be done by separate method.